include/samplers/samplers.h

```diff
@@ -4,33 +4,38 @@
 // Copyright (c) 2018 Apostolos Chalkis
 
 //Contributed and/or modified by Apostolos Chalkis, as part of Google Summer of Code 2018 program.
+//Contributed and/or modified by Repouskos Panagiotis, as part of Google Summer of Code 2019 program.
 
 // Licensed under GNU LGPL.3, see LICENCE file
 
 #ifndef RANDOM_SAMPLERS_H
 #define RANDOM_SAMPLERS_H
 
 
+
+
 // Pick a random direction as a normilized vector
 template <typename RNGType, typename Point, typename NT>
 Point get_direction(const unsigned int dim) {
 
     boost::normal_distribution<> rdist(0,1);
-    std::vector<NT> Xs(dim,0);
     NT normal = NT(0);
```

Comment:
Do you need `normal` variable here?

Reply:
You mean declare it elsewhere? Right above the loop?